packages/phone-number-privacy/signer/src/domain/domain.service.ts

```diff
@@ -118,22 +108,23 @@ export class DomainService implements IDomainService {
     request: Request<{}, {}, unknown>,
     response: Response<DomainQuotaStatusResponse>
   ): Promise<void> {
-    Counters.requests.labels(DomainEndpoint.DOMAIN_QUOTA_STATUS).inc()
+    const logger = response.locals.logger
+    const endpoint = DomainEndpoint.DOMAIN_QUOTA_STATUS
+    Counters.requests.labels(endpoint).inc()
 
     // Check that the body contains the correct request type.
     if (!domainQuotaStatusRequestSchema(DomainSchema).is(request.body)) {
-      respondWithError(
-        DomainEndpoint.DOMAIN_QUOTA_STATUS,
-        response,
-        400,
-        WarningMessage.INVALID_INPUT
-      )
+      this.sendFailureResponse(response, WarningMessage.INVALID_INPUT, 400, endpoint, logger)
+      return
+    }
+
+    if (!config.api.domains.enabled) {
+      this.sendFailureResponse(response, WarningMessage.API_UNAVAILABLE, 501, endpoint, logger)
```

Comment:
Nit: Same comment as above on 501/503
```suggestion
      this.sendFailureResponse(response, WarningMessage.API_UNAVAILABLE, 503, endpoint, logger)
```